README.md

```diff
@@ -156,8 +156,6 @@ Qi Meng, Guolin Ke, Taifeng Wang, Wei Chen, Qiwei Ye, Zhi-Ming Ma, Tie-Yan Liu.
 
 Huan Zhang, Si Si and Cho-Jui Hsieh. "[GPU Acceleration for Large-scale Tree Boosting](https://arxiv.org/abs/1706.08359)". SysML Conference, 2018.
 
-**Note**: If you use LightGBM in your GitHub projects, please add `lightgbm` in the `requirements.txt`.
```

Comment:
This was added back in #2192, around when GitHub first added the `Used By` metric (https://github.com/microsoft/LightGBM/pull/2192#discussion_r287578513) that you can view on the repo homepage.

<img width="306" alt="image" src="https://github.com/microsoft/LightGBM/assets/7608904/14a3f917-248a-4946-8a73-9764184740c0">

I think having it here could be confused as a documentation of how to use the project, and that it should be removed to simplify the instructions a bit.